mdsuite/calculators/green_kubo_ionic_conductivity.py

```diff
@@ -230,39 +223,80 @@ def _post_operation_processes(self):
         -------
 
         """
-        result = self.prefactor * np.array(self.sigma)
-
+        self.sigmas = np.array(self.sigmas)
+        sigma = np.mean(self.sigmas, axis=0)
+        sigma_SEM = np.std(self.sigmas, axis=0) / np.sqrt(len(self.sigmas))
+
+        self.acfs = np.array(self.acfs)
+        acf = np.mean(self.acfs, axis=0)
+        ionic_conductivity = self.prefactor * sigma[self.args.integration_range - 1]
+        ionic_conductivity_SEM = (
+            self.prefactor * sigma_SEM[self.args.integration_range - 1]
+        )
         data = {
-            self.result_keys[0]: np.mean(result).tolist(),
-            self.result_keys[1]: (np.std(result) / np.sqrt(len(result))).tolist(),
+            self.result_keys[0]: [ionic_conductivity],
+            self.result_keys[1]: [ionic_conductivity_SEM],
             self.result_series_keys[0]: self.time.tolist(),
-            self.result_series_keys[1]: self.jacf.numpy().tolist(),
+            self.result_series_keys[1]: acf.tolist(),
+            self.result_series_keys[2]: sigma.tolist(),
+            self.result_series_keys[3]: sigma_SEM.tolist(),
         }
 
         self.queue_data(data=data, subjects=["System"])
 
     def plot_data(self, data):
         """Plot the data"""
         for selected_species, val in data.items():
+            fig = figure(x_axis_label=self.x_label, y_axis_label=self.y_label)
+
+            integral = np.array(val[self.result_series_keys[2]])
+            integral_err = np.array(val[self.result_series_keys[3]])
+            time = np.array(val[self.result_series_keys[0]])
+            acf = np.array(val[self.result_series_keys[1]])
+            # Compute the span
             span = Span(
-                location=(
-                    np.array(val[self.result_series_keys[0]])
-                    * self.experiment.units["time"]
-                )[self.args.integration_range - 1],
+                location=np.array(val[self.result_series_keys[0]])[
+                    self.args.integration_range - 1
+                ],
                 dimension="height",
                 line_dash="dashed",
             )
-            self.run_visualization(
-                x_data=np.array(val[self.result_series_keys[0]])
-                * self.experiment.units["time"],
-                y_data=np.array(val[self.result_series_keys[1]]),
-                title=(
-                    f"{val[self.result_keys[0]]: 0.3E} +-"
-                    f" {val[self.result_keys[1]]: 0.3E}"
+            # Compute vacf line
+            fig.line(
+                time,
+                acf,
+                color="#003f5c",
```

Comment:
If we make this an MDSuite default color maybe have it as `mds.utils.colors.DARKBLUE` ?

Reply:
It's a good idea, added in #531